Buildings/ThermalZones/EnergyPlus/UsersGuide.mo

```diff
@@ -168,8 +165,7 @@ If a zone is in the idf file but not modeled in Modelica, EnergyPlus will
 simulate the zone as free floating.
 </li>
 <li>
-If there is an HVAC system in the idf file, then EnergyPlus will remove it.
-</li>
+All EnergyPlus HVAC objects that are present in the idf file are removed when coupled to Spawn</li>
```

Comment:
I think this change is very small, but in the original sentence, I did not agree with the "EnergyPlus will remove it", since that removal is done when compiling the model, independently of EnergyPlus.